pc_autobackup.py

```diff
@@ -22,6 +22,8 @@
 import ssdp
 import mediaserver
 
+import xml.etree.ElementTree as ET
```

Comment:
This belongs on line 17... imports are in stdlib, third_party, this package order... remain consistent